.buildkite/pipeline.full.yml

```diff
@@ -0,0 +1,604 @@
+        timeout_in_minutes: 30
+        agents:
+          queue: opensource
+        plugins:
+          artifacts#v1.9.3:
+            download: "features/fixtures/ios/output/bb_ipa_url.txt"
+            upload:
+              - "maze_output/failed/**/*"
+              - "maze_output/metrics.csv"
+              - "maze_output/maze_output.zip"
+          docker-compose#v4.8.0:
+            pull: maze-runner-bb
+            run: maze-runner-bb
+            service-ports: true
+            command:
+              - "--app=@build/bb_ipa_url.txt"
+              - "--device=IOS_14"
+              - "--fail-fast"
+              - "--farm=bb"
+              - "--no-tunnel"
+              - "--aws-public-ip"
+              - "features/default"
+          test-collector#v1.10.2:
+            files: "reports/TEST-*.xml"
+            format: "junit"
+            branch: "^main|next$$"
+        concurrency: 25
+        concurrency_group: bitbar-app
+        concurrency_method: eager
+
+      - label: ":bitbar: iOS 13 E2E Tests"
+        depends_on:
+          - ios_fixture
+        timeout_in_minutes: 30
+        agents:
+          queue: opensource
+        plugins:
+          artifacts#v1.9.3:
+            download: "features/fixtures/ios/output/bb_ipa_url.txt"
+            upload:
+              - "maze_output/failed/**/*"
+              - "maze_output/metrics.csv"
+              - "maze_output/maze_output.zip"
+          docker-compose#v4.8.0:
+            pull: maze-runner-bb
+            run: maze-runner-bb
+            service-ports: true
+            command:
+              - "--app=@build/bb_ipa_url.txt"
+              - "--device=IOS_13"
+              - "--fail-fast"
+              - "--farm=bb"
+              - "--no-tunnel"
+              - "--aws-public-ip"
+              - "features/default"
+          test-collector#v1.10.2:
+            files: "reports/TEST-*.xml"
+            format: "junit"
+            branch: "^main|next$$"
+        concurrency: 25
+        concurrency_group: bitbar-app
+        concurrency_method: eager
+
+  - group: "XcFramework E2E Tests Full"
+    steps:
+      - label: ":browserstack: iOS XcFramework 17 E2E Tests"
+        depends_on:
+          - ios_xcframework_fixture
+        timeout_in_minutes: 30
+        agents:
+          queue: opensource
+        plugins:
+          artifacts#v1.9.3:
+            download: "features/fixtures/ios/output/bs_xcframework_ipa_url.txt"
+            upload:
+              - "maze_output/failed/**/*"
+              - "maze_output/metrics.csv"
+              - "maze_output/maze_output.zip"
+          docker-compose#v4.8.0:
+            pull: maze-runner-bs
+            run: maze-runner-bs
+            service-ports: true
+            command:
+              - "--app=@build/bs_xcframework_ipa_url.txt"
+              - "--device=IOS_17"
+              - "--fail-fast"
+              - "--farm=bs"
+              - "--exclude=features/default/automatic_spans_generic_view_load.feature"
```

Comment:
`--exclude` isn't needed as we are only running a single feature anyway.  Same in a few other places too.